tests/meta-tag-tests.js

```diff
@@ -0,0 +1,208 @@
+/* eslint no-unused-expressions: "off" */
+const chai = require('chai');
+const yaml = require('js-yaml');
+const fs = require('fs');
+const path = require('path');
+const childProcess = require('child_process');
+const jsdom = require('jsdom');
+
+const { JSDOM } = jsdom;
+const {expect} = chai;
```

Comment:
Question or nitpick:  I'm a JS novice, does the space matter for this syntax?  Is there a reason one has spaces and the other doesn't?